assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/index.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames';
+import { format, isValid } from 'date-fns';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './style.css';
+import EspressoButton from '@appInputs/EspressoButton';
+
+export interface EditButtonProps {
+	tooltip: string;
+	tooltipPosition: string;
+}
+
+export enum Position {
+	LEFT = 'left',
+	RIGHT = 'right',
+}
+
+export interface BiggieCalendarDateProps {
+	date: Date;
+	editButton: EditButtonProps;
+	htmlClass: string;
```

Comment:
As mentioned in [this comment](https://github.com/eventespresso/event-espresso-core/pull/2366#discussion_r378202670), we should decide about a standard.